wordpress-mobile/WordPress-Android
# Issue/13434 added missing comment related analytics

Fixes #13434

A couple of events slipped through the cracks, so I'm adding them here:
- Batch comment editing.
- Opening of comment editor.
- Separate tracking of comments on a Reader post and replies to readers post a comment.

I also removed the `COMMENT_REPLIED_TO` tracking from Reader Post comments.

To test:
- Place breakpoints at the new trackers and confirm that they are getting hit when the corresponding action is performed.

PR submission checklist:
